draft-ietf-httpbis-h3-websockets.md

```diff
@@ -71,6 +71,10 @@ in HTTP/2 as defined {{!RFC8441}}. {{Appendix A.3 of HTTP3}} requires that
 HTTP/3 settings be registered separately for HTTP/3. The
 SETTINGS_ENABLE_CONNECT_PROTOCOL value is 0x08 (decimal 8), as in HTTP/2.
 
+If a server receives an Extended CONNECT request with a ":protocol"
+value that is unknown or is not supported, the server SHOULD respond
+to the request with a 501 status code, see {{Section 6.6.2 of RFC7231}}.
```

Comment:
The build failure is this reference. FWIW you probably want to point to https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#status.501 and you could do so by adding `HTTP` to your normative refs and then using that here.

Reply:
Stil lthink you're better off target RFC-to-be replacement for RFC7231 :D

Reply:
Thanks! Sorry, I totally missed your comment when I was fixing the build error. Hopefully I got it right this time! :)

Reply:
heh 409 conflict ;)